x-pack/plugin/security/qa/operator-privileges-tests/build.gradle

```diff
@@ -39,7 +39,6 @@ testClusters.configureEach {
   setting 'path.repo', repoDir.absolutePath
 
   requiresFeature 'es.user_profile_feature_flag_enabled', Version.fromString("8.1.0")
-  requiresFeature 'es.rollup_v2_feature_flag_enabled', Version.fromString("8.0.0")
```

Comment:
Should this have the time series flag instead?

Reply:
Fixed in [270ea69](https://github.com/elastic/elasticsearch/pull/85708/commits/270ea69604e30fc6f3e1fc6993f9ddf8d84c91ba)